README.md

```diff
@@ -155,7 +180,7 @@ Cloud Storage and generate the XML, which is then deployed with the Java binary.
 
 Server behavior configuration is stored in the database. It may be changed
 without restarting the server, but is usually set only at deployment time. It's
-based on config_$ENV.json files (which are converted into `WorkbenchConfig`
+based on config\_$ENV.json files (which are converted into `WorkbenchConfig`
```

Comment:
This looks the same both ways to me. You're guarding against accidental italics? (Seems fine, just curious.)

Reply:
yeah - that might just be an accident of my editor's syntax highlighting - everything after this underscore was italicized while I was editing, so I escaped it.

Reply:
You could also try `config_$ENV.json` (using backticks).